jobs/keepalived/templates/bin/job_properties.sh.erb

```diff
@@ -28,4 +28,16 @@ IS_HA= link("broker").p("ha_enabled")
 
     export NETWORK="-network=<%= spec.networks.sf_service_fabrik_broker.cloud_properties.network_name %>"
     export SUBNET="-subnetwork=<%= spec.networks.sf_service_fabrik_broker.cloud_properties.subnetwork_name %>"
+<% end %>
+<% if iaas_name == "openstack" && IS_HA == true %>
+  export OS_AUTH_URL="<%= p('iaas_config.auth_url') %>"
+  export OS_AUTH_VERSION="3"
+  export OS_IDENTITY_API_VERSION="3"
+  export OS_USER_DOMAIN_NAME="<%= p('iaas_config.user_domain_name') %>"
+  export OS_USERNAME="<%= p('iaas_config.username') %>"
+  export OS_PASSWORD="<%= p('iaas_config.password') %>"
+  export OS_PROJECT_ID="<%= p('iaas_config.tenant_id') %>"
+  export OS_PROJECT_NAME="<%= p('iaas_config.tenant_name') %>"
+  export OS_REGION_NAME="<%= p('iaas_config.region') %>"
+  export FLOATING_IP="<%= link('broker').p('internal.ip') %>"
```

Comment:
Just a question: is it possible that we get an ambiguous bosh link reference here? Wouldn't it be better to be explicit in the linking?

Reply:
Skipping this as discussed. With name 'broker' there would be only one link. Will take care if there is any ambiguity in future.